communities/src/tests.rs

```diff
@@ -210,7 +211,7 @@ fn new_community_works() {
             None,
             None
         ));
-        let cid = CommunityIdentifier::from(blake2_256(&(location.clone(), bs.clone()).encode()));
+        let cid = create_community_identifier(location.clone(), &(bs.clone()).encode());
```

Comment:
It would be more comfortable to just pass `bs` and do the encoding within the fn

and I'd rather have a constructor `CommunityIdentifier::new()` than a helper function

Reply:
The problem I had there is, that bs is a Vec<T::AccountId> and this is not known around the context of communities.rs. Using a completely generic vector parameter inhibits the use of encode(). Maybe this is an issue, that you or @clangenb could quickly fix/explain. :)

Reply:
We can just pass the trait bound `AccountId: Encode`, which will always be fulfilled for the actual `T::AccountId`